web_m2x_options/static/src/js/form.js

```diff
@@ -21,6 +21,47 @@ odoo.define('web_m2x_options.web_m2x_options', function (require) {
         method: 'get_web_m2x_options',
     });
 
+    function is_option_set(option){
+        if (_.isUndefined(option))
+            return false;
+        if (typeof option === 'string')
+            return option === 'true' || option === 'True';
+        if (typeof option === 'boolean')
+            return option;
+        return false
+    };
+
+    var settings;
+
+    web_m2x_options.done(function(res) {
```

Comment:
So I suppose that by moving this query at the JS module level will trigger the query when the JS client is loaded? Make sense after all.

Reply:
Checked this, and it does.